extensions/mongodb-client/deployment/src/main/java/io/quarkus/mongodb/deployment/MongoClientBuildItem.java

```diff
@@ -3,7 +3,7 @@
 import com.mongodb.client.MongoClient;
 
 import io.quarkus.builder.item.MultiBuildItem;
-import io.quarkus.mongodb.ReactiveMongoClient;
+import io.quarkus.mongodb.reactive.ReactiveMongoClient;
```

Comment:
Shouldn't it contains the both reactive clients ?

Reply:
Except if it's required for Panache, it would be preferable that extensions do not use the deprecated client as it's planed for removal.

Reply:
I don't remember all the discussions but I think the purpose of this BuildItem is to allow extensions depending on `mongodb-client` (today, only panache) to acess them.
If we decide that panache should only provides an implementation based on Mutiny (OK for me as the implementation based on Axle didn't exist in any publicly available Quarkus version), I'm OK with including only one reactive client.